charts/vc-authn-oidc/values.yaml

```diff
@@ -607,7 +604,7 @@ mongodb:
     ## If undefined (the default) or set to null, no storageClassName spec is
     ## set, choosing the default provisioner.
     ##
-    storageClass: netapp-block-standard
+    storageClass: default
```

Comment:
I believe this is correct, and a different storage provider can be specified if necessary by each deployer